sessions/class-01/exercise/start.js

```diff
@@ -22,8 +32,10 @@ export function safelyConvertToNumber(possiblyNum) {}
  * @param {number} decimal Decimal precision.
  * @returns {number} Formatted currency value.
  */
-export function formatCurrency(num, decimal) {}
-
+export function formatCurrency(num, decimal) {
+	return safelyConvertToNumber(num).toFixed(decimal);
```

Comment:
This returns a `string`. But we need to return a `number` from the function.

So we can

```js
return Number(safelyConvertToNumber(num).toFixed(decimal));
```

Or just

```js
	return +safelyConvertToNumber(num).toFixed(decimal);
```